### PR TITLE
fix(TBD-7933): fix missing talend-bigdata-launcher-1.2.0-20170410.jar

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Comment by sponomarova** [comment]
Seems like BIGDATALAUNCHER_MODULE_GROUP was not included into HDInsight36SparkBatchModuleGroup. 
I updated PR to address this issue.

File: main/plugins/org.talend.hadoop.distribution.hdinsight360/src/main/java/org/talend/hadoop/distribution/hdinsight360/modulegroup/HDInsight36SparkBatchModuleGroup.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     public static Set<DistributionModuleGroup> getModuleGroups() {
         Set<DistributionModuleGroup> hs = new HashSet<>();
         hs.add(new DistributionModuleGroup(HDInsight36Constant.SPARK_MODULE_GROUP.getModuleName(), false, condition));
-        hs.add(new DistributionModuleGroup(HDInsight36Constant.BIGDATALAUNCHER_MODULE_GROUP.getModuleName(), true, condition));
```
Comment (yu-talend) [inline main/plugins/org.talend.hadoop.distribution.hdinsight360/src/main/java/org/talend/hadoop/distribution/hdinsight360/modulegroup/HDInsight36SparkBatchModuleGroup.java]: The last 2 arguments only control the jar is `MRRequired` or not and if it meets the "not local spark" condition. So under these requirements the jar should be loaded ?

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdinsight360/src/main/java/org/talend/hadoop/distribution/hdinsight360/modulegroup/HDInsight36SparkBatchModuleGroup.java]: Seems like 

`hs.add(new DistributionModuleGroup(HDInsight36Constant.BIGDATALAUNCHER_MODULE_GROUP.getModuleName(), true, condition));
`
should be loaded either with MRRequired = false or without additional conditions as it is done in PR.

jar is loaded from di job only under these conditions.

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdinsight360/src/main/java/org/talend/hadoop/distribution/hdinsight360/modulegroup/HDInsight36SparkBatchModuleGroup.java]: Similar way it is implemented for Qubole distro:
`public class QuboleSparkBatchModuleGroup {

    public static Set<DistributionModuleGroup> getModuleGroups() {
        Set<DistributionModuleGroup> moduleGroups = new HashSet<>();
        ...
        moduleGroups.add(new DistributionModuleGroup(QuboleConstant.BIGDATALAUNCHER_MODULE_GROUP.getModuleName()));
        ...
        return moduleGroups;
    }`

**Pull request merged**